combat-trainer.lic

```diff
@@ -189,12 +190,43 @@ class LootProcess
   def dispose_body(game_state)
     return if DRRoom.dead_npcs.length < 1
     game_state['mob_died'] = true
-
-    check_skinning(DRRoom.dead_npcs.first) if @skin
+    
+#NECRO THANATOLOGY CHECK STUFF GOES BELOW
+    
+    if @necro {check_thanatology(DRRoom.dead_npcs.first)}
+      dispose_body_loot
+      
+#NECRO THANATOLOGY CHECK STUFF GOES ABOVE
+    
+    elseif @skin { check_skinning(DRRoom.dead_npcs.first) }
+      dispose_body_loot
```

Comment:
For clarity I'd use if end blocks, so 

if @necro
  check_thanatology(DRRoom.dead_npcs.first)
elsif @skin
  check_skinning(DRRoom.dead_npcs.first)
end
dispose_body_loot

You're calling dispose loot twice right now, so just move it outside of the if statement since you want to call it regardless.